qemu/tests/cfg/blockdev_mirror_to_rbd.cfg

```diff
@@ -30,3 +29,7 @@
     enable_ceph_mirror1 = yes
     enable_ceph_data1 = no
     enable_ceph_image1 = no
+    ceph:
```

Comment:
Please run this case with filesystem and ceph backends, check the logs to see if the qmp commands are expected.